### PR TITLE
Added React Hooks

### PR DESCRIPTION
**Pull request by hschaen**

### CODE REVIEW

File: cat-tinder/frontend/cat-index.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ render(){
 
 Test should pass because we have a component that can be rendered. It imports React and has a render function, it does not yet show real content.
 
+
```
Comment (sjproctor) [inline cat-tinder/frontend/cat-index.md:49]: Render on line 35 should be removed.

File: cat-tinder/frontend/cat-index.md [hunk 2]
```diff
@@ -79,7 +80,7 @@ export default cats
 
 The fake cats variable can be passed into the state object in App.js.
 
-```
+```JavaScript
```
Comment (sjproctor) [inline cat-tinder/frontend/cat-index.md:83]: This code snippet still has a class component, a dynamic route, and props.

File: cat-tinder/frontend/cat-index.md [hunk 3]
```diff
@@ -150,6 +151,8 @@ The tests should now pass.
 
 [Go to next lesson: Cat Tinder Create New Cat Component](./cat-create.md)
 
+[Go to next lesson: Cat Tinder React Hooks](./cat-create.md)
+
```
Comment (sjproctor) [inline cat-tinder/frontend/cat-index.md:155]: The New Cat link should be removed.

**Comment by orangewolf** [comment]
@hschaen this one needs to be updated before Monday morning so I'm making the edits.

File: cat-tinder/frontend/cat-index.md (hunk 3 above)
Comment (orangewolf) [reply cat-tinder/frontend/cat-index.md:153]: 👍

**Pull request merged**